static/gcc.js

```diff
@@ -292,11 +293,22 @@ function loadState(state) {
     setFilterUi(state.filterAsm);
     for (var i = 0; i < Math.min(allCompilers.length, state.compilers.length); i++) {
         allCompilers[i].setFilters(state.filterAsm);
-        allCompilers[i].deserialiseState(state.compilers[i]);
+        allCompilers[i].deserialiseState(state.compilers[i],OPTIONS.compilers, OPTIONS.defaultCompiler);
     }
     return true;
 }
 
+function resizeEditors() {
```

Comment:
Thanks for clarifications,
I agree, we have to choose what we want the UI to look like.